### DISCUSSION TITLE
BG: Can you use both the Black Gate Keyring and Nitpicker's Patch together?

### DISCUSSION THREAD

**Discussion by Zhelkas** [exult/exult#374] [General]
Hey everyone,

I love the Black Gate Keyring mod for U7:BG.  Playing through a game of it right now.  My question is, is it possible to have both this mod and the fixes from the Nitpicker's Patch combined, so I can have the best of both worlds?

Thanks in advance!

**Comment by DominusExult** [comment]
It *might* be possible.
@ungesundes-halbwissen's patch provides a complete patched USECODE file, if you replace the original one with that (making a backup and then copying the nitpicker's USECODE file) and then using the Keyring mod *might* work. One would need to investigate the other files. Probably also by replacing them.
I'd advise against the General Midi patch, though. That recently caused a problem with our Android port (see #369).

**Reply by Zhelkas** [reply]
Thanks!  I'll give it a shot and let you know if it works.

**Reply by Zhelkas** [reply]
So far it seems to work!  Just copying over the USECODE file seems to have done the trick.  I backed up the old one in case any issues come up, but so far I haven't noticed any problems, and the Nitpickers Delight fixes are all working.

**Comment by ungesundes-halbwissen** [comment]
Letting us know, if it works would be highly appreciated, thanks in advance. If it needed fiddling around, please share as well, what you needed to do, to get it up and running. Chances are, you will have to use only some of the Nitpickers binaries.

**Reply by Zhelkas** [reply]
Thanks.  So far it looks like it's working fine with @DominusExult's suggestion.  I copied over the USECODE file (after backing up the old one) and the Nitpickers Delight fixes are all there on top of the Black Gate Keyring mod.
